lib/hanami/slice.rb

```diff
@@ -99,17 +99,20 @@ def define_container
       container = Class.new(Dry::System::Container)
       container.use :env
 
-      container.config.name = name
-      container.config.inflector = application.configuration.inflector
-
-      if root && File.directory?(root)
-        container.config.root = root
-        container.config.bootable_dirs = ["config/boot"]
-
-        container.config.auto_register = ["lib/#{namespace_path}"]
-        container.config.default_namespace = namespace_path.tr("/", ".")
-
-        container.load_paths! "lib"
+      container.configure do |config|
+        config.name = name
+        config.inflector = application.configuration.inflector
+
+        if root && File.directory?(root)
```

Comment:
Just a minor thing, shouldn't we check `root.directory?` as we do [here](https://github.com/hanami/hanami/pull/1093/files#diff-821c48c88c392a3a4fb873393e0c3c8401ca598885b47e1cfe8c0d4c57932293R253)?